kotlin-grpc/src/main/kotlin/com/linecorp/armeria/server/grpc/CoroutineServerInterceptor.kt

```diff
@@ -0,0 +1,81 @@
+package com.linecorp.armeria.server.grpc
```

Comment:
It is not recommended to share the same package with different modules.
```suggestion
package com.linecorp.armeria.server.grpc.kotlin
```
As `AbstractServerCall` is package-private, we can either 1) move `AbstractServerCall` to the `internal` module or 2) use reflections to access the methods unsafely.
What do you think?

Reply:
I think the former one seems better.
Can you guide me the exact directory of `internal` module you mentioned?

Reply:
I moved `AbstractServerCall` to internal package in grpc module, and modified some access modifiers 🙂